ws-nextjs-app/pages/[service]/send/[id]/FormContext/index.tsx

```diff
@@ -115,14 +141,22 @@ export const FormContextProvider = ({
 
       req.send(formData);
     } catch (error) {
+      console.log(error);
```

Comment:
If we mean to keep this, can we use `error` instead of `log`
```suggestion
      console.error(error);
```

Reply:
ah sorry, meant to remove that